armTemplates/azuredeploy-blobforwarder.json

```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newRelicLicenseKey": {
+      "type": "string",
+      "metadata": {
+        "description": "New Relic License Key"
+      }
+    },
+    "newRelicEndpoint": {
+      "type": "string",
+      "defaultValue": "https://log-api.newrelic.com/log/v1",
+      "metadata": {
+        "description": "The Logs API endpoint used to send your logs to. By default, it is https://log-api.newrelic.com/log/v1 if your account is in the United States (US) region. Otherwise, if you're in the European Union (EU) region, you should use https://log-api.eu.newrelic.com/log/v1"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Azure Storage account."
+      },
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "storageAccountSku": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Standard_ZRS",
+        "Premium_LRS",
+        "Premium_ZRS",
+        "Standard_GZRS",
+        "Standard_RAGZRS"
+      ],
+      "metadata": {
+        "description": "Sku on which to run the Azure Storage account."
+      }
+    },
+    "storageAccountKind": {
+      "type": "string",
+      "defaultValue": "StorageV2",
+      "allowedValues": [
+        "Storage",
+        "StorageV2",
+        "BlobStorage",
+        "FileStorage",
+        "BlockBlobStorage"
+      ],
+      "metadata": {
+        "description": "Indicates the type of storage account."
+      }
+    },
+    "storageAccountLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location where the storage account reside. (eg. Central US)"
+      }
+    },
+    "storageAccountContainer": {
+      "type": "string",
+      "metadata": {
+        "description": "A container in the storage account"
+      },
+      "minLength": 3,
+      "maxLength": 63
+    }
+  },
+  "variables": {
+    "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]",
+    "blobForwarderFunctionArtifact": "https://github.com/newrelic/newrelic-azure-functions/releases/latest/download/BlobForwarder.zip",
+    "onePerResourceGroupAndStorageAccountAndContainer": "[uniqueString(resourceGroup().id, parameters('storageAccountName'), parameters('storageAccountContainer'))]",
+    "functionAppName": "[concat('nrlogs-blobforwarder-', variables('onePerResourceGroupAndStorageAccountAndContainer'))]"
```

Comment:
❔ curiosity, to understand our solution
> onePerResourceGroupAndStorageAccountAndContainer

Let's say one customer want to forward same logs to diff account, unless function is in a diff resource group, is that possible?

Reply:
No, inside the same resource group it's not possible! Even it was thought as a way to minimize customers mistake and send the same log many times.